unity/Assets/Scripts/InstantiatePrefabTest.cs

```diff
@@ -433,6 +433,11 @@ bool AlwaysPlaceUpright
                 // set true if we want objects to be stationary when placed. (if placed on uneven surface, object remains stationary)
                 // if false, once placed the object will resolve with physics (if placed on uneven surface object might slide or roll)
                 if (PlaceStationary == true) {
```

Comment:
Can't this just be `if (PlaceStationary)`?

Reply:
yup, will tidy this up